internal/util/kubernetes/unstructured_util.go

```diff
@@ -53,7 +53,7 @@ func GetLiveResource(
 		return nil, err
 	}
 
-	unstruc, err := dynamicClient.Resource(gvr).Namespace(object.GetNamespace()).Get(ctx, object.GetName(), metav1.GetOptions{})
+	unstruc, err := DynamicClient.Resource(gvr).Namespace(object.GetNamespace()).Get(ctx, object.GetName(), metav1.GetOptions{})
```

Comment:
Do we have plan to use "DynamicClient" outside kubernetes package?

Reply:
To me, it seems okay if we were to. And it seems weird to only expose `KubernetesClient` and not `DynamicClient`. I don't really want to be constrained that only the `kubernetes` package can use it. Do you disagree?